assets/scripts/script.js

```diff
@@ -58,8 +58,7 @@
         thumbnail: true,
         showThumbByDefault: false,
         subHtmlSelectorRelative: true,
-        selector: '.woocommerce-product-gallery__image > a',
-        width: '1010px'
```

Comment:
Did you check the commit history (`git log -G1010px`) to see when and why this was introduced?

Reply:
I introduced the width myself. It was my first fix to fix the enlarging images issue, but I didn't consider the fact that smaller images would be also be enlarged up to 1010px, so, the fix is useless.